app/_partner/factset.md

```diff
@@ -51,6 +51,7 @@ subhashtags:
   - factset20singapore
   - factset20sydney
   - factset20tokyo
+  - factsetgermany
```

Comment:
should this be `factset20germany`?

Reply:
Weirdly not, they had a mapathon today that mistakenly used this old hashtag and now they want to see how it compares

Reply:
ok cool. merging now

Reply:
Many thanks Dakota!